README.md

```diff
@@ -53,6 +57,51 @@ Here is an example command line to launch the tool:
 $ dockstore tool launch --entry  quay.io/briandoconnor/dockstore-tool-md5sum  --json test.gs.json
 ```
 
+## Releases
+
+This section describes creating a release of the Google Cloud Storage file provisioning plugin.
+
+### Prerequisites
+
+[Install](https://datasift.github.io/gitflow/TheHubFlowTools.html) Hubflow. After it is installed, run `git hubflow init` in the 
+root of your copy of the repo.
+
+### Create the Release
+
+We will be creating a 0.0.4 release as an example.
+
+1. `git hf release start <version number>` -- Creates a new branch based off develop. With `0.0.4` as the version number, the new
+branch will be release/0.0.4.
+2. `mvn release:prepare` -- Prompts you to:
```

Comment:
Not a requirement but for markdown as long as you are using any number in a list the numbers will be in correct order. So if you just number everything 1. it will make it easier to update.